packages/fx-core/package.json

```diff
@@ -105,7 +105,7 @@
     "@feathersjs/hooks": "^0.6.5",
     "@microsoft/dev-tunnels-contracts": "1.1.9",
     "@microsoft/dev-tunnels-management": "1.1.9",
-    "@microsoft/kiota": "1.25.1-preview.202504240001",
+    "@microsoft/kiota": "1.25.1-preview.202505010001",
```

Comment:
@SLdragon @sebastienlevert  I believe we need to test the features first before updating this version.

Reply:
They seem related to the latest updates to the flow from @SLdragon.